code/game/objects/items/stacks/wrap.dm

```diff
@@ -9,11 +9,48 @@
 	desc = "Wrap packages with this festive paper to make gifts."
 	icon = 'icons/obj/stack_objects.dmi'
 	icon_state = "wrap_paper"
+	greyscale_config = /datum/greyscale_config/wrap_paper
 	item_flags = NOBLUDGEON
 	amount = 25
 	max_amount = 25
 	resistance_flags = FLAMMABLE
 	merge_type = /obj/item/stack/wrapping_paper
+	var/base_color = "#00FF00"
+	var/ribbon_color = "#FF0000"
```

Comment:
Is it just me, or are these defaults never used, unless `greyscale_colors` is set, in which case they might not correspond to the `greyscale_colors` var?

Reply:
I could leave them undefined, but they need to be accessible during the wrapping process otherwise I could have left it all within the block below. It's mostly just a fail-safe?

Reply:
I don't know that much about GAGS, but is it possible to "reverse" the set_greyscale call? So instead of having a `base_color` and `ribbon_colour` var, it just extracts that from the greyscale settings?

Reply:
Yeah it turns out I can pass `greyscale_colors` to the `set_greyscale` on the packages, so I cut all that out.